x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/cti/threat_match_row.tsx

```diff
@@ -45,12 +45,12 @@ export const ThreatMatchRow = ({
   const props = {
     contextId,
     eventId,
-    indicatorReference: get(data, REFERENCE)[0] as string | undefined,
-    indicatorProvider: get(data, PROVIDER)[0] as string | undefined,
-    indicatorType: get(data, MATCHED_TYPE)[0] as string | undefined,
+    indicatorReference: getOr([], REFERENCE, data)[0] as string | undefined,
+    indicatorProvider: getOr([], PROVIDER, data)[0] as string | undefined,
+    indicatorType: getOr([], MATCHED_TYPE, data)[0] as string | undefined,
     isDraggable,
-    sourceField: get(data, MATCHED_FIELD)[0] as string,
-    sourceValue: get(data, MATCHED_ATOMIC)[0] as string,
+    sourceField: getOr([], MATCHED_FIELD, data)[0] as string,
+    sourceValue: getOr([], MATCHED_ATOMIC, data)[0] as string,
```

Comment:
Line 52- 52, If we put the fallback value as empty an array, means that eventually it could still be undefined here, right?

Reply:
Good catch! Addressed in dc797dfb859.